.mailmap

```diff
@@ -1,42 +1,46 @@
 Abdul Mutaal Ahmad <abdul.mutaal@gmail.com>
 adarshpatil <adarshpatil123@gmail.com>
+Aditya K Kamath <a_kamath@hotmail.com> aditya <a_kamath@hotmail.com>
 Adrià Armejach <adria.armejach@bsc.es> Adrià Armejach <adria.armejach@gmail.com>
+Adrià Armejach <adria.armejach@bsc.es> Adrià Armejach <66964292+aarmejach@users.noreply.github.com>
 Adrian Herrera <adrian.herrera@arm.com>
 Adrien Pesle <adrien.pesle@arm.com>
+Adwaith R Krishna <adwaithrk19@gmail.com>
 Akash Bagdia <akash.bagdia@ARM.com> Akash Bagdia <akash.bagdia@arm.com>
 Alec Roelke <alec.roelke@gmail.com> Alec Roelke <ar4jc@virginia.edu>
 Alexander Klimov <Alexander.Klimov@arm.com>
 Alexandru Dutu <alexandru.dutu@amd.com> Alexandru <alexandru.dutu@amd.com>
 Alex Richardson <alexrichardson@google.com>
 Ali Jafri <ali.jafri@arm.com>
 Ali Saidi <Ali.Saidi@arm.com> Ali Saidi <ali.saidi@arm.com>
-Ali Saidi <Ali.Saidi@arm.com> Ali Saidi <Ali.Saidi@ARM.com>
 Ali Saidi <Ali.Saidi@arm.com> Ali Saidi <saidi@eecs.umich.edu>
 Alistair Delva <adelva@google.com>
+Alvaro Moreno <alvaro.moreno@bsc.es>
 Amin Farmahini <aminfar@gmail.com>
 Anders Handler <s052838@student.dtu.dk>
 Andrea Mondelli <andrea.mondelli@huawei.com> Andrea Mondelli <andrea.mondelli@ucf.edu>
-Andrea Mondelli <andrea.mondelli@huawei.com> Andrea Mondelli <Andrea.Mondelli@ucf.edu>
 Andrea Pellegrini <andrea.pellegrini@gmail.com>
 Andreas Hansson <andreas.hanson@arm.com> Andreas Hansson <andreas.hansson>
 Andreas Hansson <andreas.hanson@arm.com> Andreas Hansson <andreas.hansson@arm.com>
-Andreas Hansson <andreas.hanson@arm.com> Andreas Hansson <Andreas.Hansson@ARM.com>
 Andreas Hansson <andreas.hanson@arm.com> Andreas Hansson <andreas.hansson@armm.com>
 Andreas Sandberg <Andreas.Sandberg@arm.com> Andreas Sandberg <andreas.sandberg@arm.com>
-Andreas Sandberg <Andreas.Sandberg@arm.com> Andreas Sandberg <Andreas.Sandberg@ARM.com>
```

Comment:
I _think_ there were reasons we kept these capitalizations. Some commits had them and we needed to include these entries to match them (there was some case-sensitivity in the way the author search works in git, maybe (??)). 

Have we proven they are unnecessary?

Reply:
It looks like the mailmap file works case-insensitively. You can check it out [here](https://git-scm.com/docs/gitmailmap).

Reply:
It's worth checking if this actually works the way it's specified in the documentation. I tried removing the different capitalisations locally at one point and it seemed like the `git-shortlog` didn't merge entries correctly at the time.

Reply:
We did check before pushing the commit that the file works case-insensitively. Maybe it didn't work that way before.